scripts/!mods_preload/msu_debug.nut

```diff
@@ -206,43 +151,35 @@ gt.MSU.setupDebuggingUtils <- function()
 				}
 			}
 		}
-		MSUMainDebugFlag = {
-			debug = true
-		}
-
-		MSUDebugFlags = {
-			movement = true,
-			skills = false,
-			keybinds = false,
-			persistence = true
-		}
 	}
 
-	::printLog <- function( _arg, _modID, _flagID = this.MSU.Debug.DefaultFlag)
+	this.MSU.Systems.Log <- this.MSU.Class.LogSystem();
+
+	::printLog <- function( _arg, _modID, _flagID = this.MSU.Systems.Log.DefaultFlag)
 	{
-		this.MSU.Debug.print(_arg, _modID, this.MSU.Debug.LogType.Info, _flagID);
+		this.MSU.Systems.Log.print(_arg, _modID, this.MSU.Systems.Log.LogType.Info, _flagID);
 	}
 
-	::printWarning <- function( _arg,  _modID, _flagID = this.MSU.Debug.DefaultFlag)
+	::printWarning <- function( _arg,  _modID, _flagID = this.MSU.Systems.Log.DefaultFlag)
 	{
-		this.MSU.Debug.print(_arg, _modID, this.MSU.Debug.LogType.Warning, _flagID);
+		this.MSU.Systems.Log.print(_arg, _modID, this.MSU.Systems.Log.LogType.Warning, _flagID);
 	}
 
-	::printError <- function( _arg,  _modID, _flagID = this.MSU.Debug.DefaultFlag)
+	::printError <- function( _arg,  _modID, _flagID = this.MSU.Systems.Log.DefaultFlag)
 	{
-		this.MSU.Debug.print(_arg, _modID, this.MSU.Debug.LogType.Error, _flagID);
+		this.MSU.Systems.Log.print(_arg, _modID, this.MSU.Systems.Log.LogType.Error, _flagID);
 	}
 
-	::isDebugEnabled <- function(_modID, _flagID = this.MSU.Debug.DefaultFlag){
-		return this.MSU.Debug.isEnabledForMod( _modID, _flagID)
+	::isDebugEnabled <- function(_modID, _flagID = this.MSU.Systems.Log.DefaultFlag){
```

Comment:
If we are going with `this.MSU.Systems.Log` then the name of the shortcut needs to be updated perhaps to something like ``::isLogEnabled`